### PR TITLE
DDF-3640 icon change, ordering, whitespace, and vertical dividers

### PR DESCRIPTION
**Pull request by GabrielFabian**
#### What does this PR do?
Improves action toolbar as per ticket DDF-3640.

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@lcrosenbu @bdeining @jlcsmith @andrewkfiedler 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler
@bdeining

#### What are the relevant tickets?
[DDF-3640](https://codice.atlassian.net/browse/DDF-3640)

#### Screenshots (if appropriate)
screen shot 2018-02-22 at 11 18 00 am

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.hbs [hunk 1]
```diff
@@ -11,17 +11,17 @@
  *
  **/
  --}}
-<button class="navigation-item item-help is-button" title="Shows helpful hints in the current context.">
-    <span class="fa fa-question"></span>
+<button class="navigation-item navigation-icon item-help is-button" title="Shows helpful hints in the current context.">
+    <span class="fa fa-question-circle"></span>
```
Comment (andrewkfiedler) [inline catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.hbs]: Why the change to the circle?

File: catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.less [hunk 2]
```diff
@@ -16,6 +16,13 @@
         width: auto;
         overflow: hidden;
         text-overflow: ellipsis;
+        margin-right: 9rem;
```
Comment (andrewkfiedler) [inline catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.less]: We should use the vars defined in `vars.less`.

That includes `@minimumSpacing`, `@mediumSpacing`, and `@largeSpacing`.

File: catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.less [hunk 3]
```diff
@@ -16,6 +16,13 @@
...
+    }
+
+    .navigation-item.navigation-icon {
+        margin-right: 0.5em;
+        padding-left: 0.575em;
```
Comment (andrewkfiedler) [inline catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.less]: This is not going to be a good idea.  The border doesn't help out, and the margin creates a deadzone for clicks.  In addition, the padding will make it so the icon isn't centered.

**Comment by andrewkfiedler** [comment]
screen shot 2018-02-22 at 12 23 52 pm

**Comment by andrewkfiedler** [comment]
Instead of all the padding & margin changes, update the width of the buttons instead (add an extra minimum or medium spacing).

File: catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.hbs (hunk 1 above)
Comment (GabrielFabian) [reply catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.hbs]: Just the way the requirement was passed down to me in the ticket. I believe it conforms to the way-forward UX mockups from @TeresaHudson and @michaelmenousek .
